### PR TITLE
Added suggestions from PR6068

### PR DESCRIPTION
**Pull request by StevenPuttemans**
As discussed here: https://github.com/Itseez/opencv/pull/6068#issuecomment-180797723

### CODE REVIEW

**Comment by StevenPuttemans** [comment]
@mshabunin as requested :D

File: modules/core/include/opencv2/core/mat.hpp [hunk 1]
```diff
@@ -1706,9 +1706,12 @@ class CV_EXPORTS Mat
     Keep in mind that the size identifier used in the at operator cannot be chosen at random. It depends
     on the image from which you are trying to retrieve the data. The table below gives a better insight in this:
      - If matrix is of type `CV_8U` or `CV_8S` then use `Mat.at<uchar>(y,x)`.
```
Comment (mshabunin) [inline modules/core/include/opencv2/core/mat.hpp]: `CV_8S` is left here.

**Comment by StevenPuttemans** [comment]
Fixed, but on browers interface so no way to rebase easily and merge commits ...

**Comment by mshabunin** [comment]
:+1:

**Pull request merged**